pkg/migration/controllers/migration.go

```diff
@@ -614,6 +705,20 @@ func (m *MigrationController) migrateResources(migration *stork_api.Migration) e
 	if err != nil {
 		return err
 	}
+	logrus.Infof("Cleaning up migrated resources")
+	// delete resources on destination which are not present on source
+	// TODO: what should be idle location for this
```

Comment:
This should be before calling `sdk.Update()` above. Otherwise if stork crashes after updating the object the cleanup will never get called on restart

Reply:
I had sdk.Update seperately in `cleanupMigratedResources` since proper location of cleanup was not decided, but we should have only one sdk.update as you suggested